web-app/src/pages/Dashboard/container.js

```diff
@@ -2,30 +2,32 @@ import React from 'react';
 import { PropTypes as PT } from 'prop-types';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
-import { fetchEvents, fetchEventsByUserId } from '../../actions/dashboard';
+import { fetchEvents } from '../../actions/dashboard';
 import { getUser, getTakenHolidays, eventBeingUpdated } from '../../reducers';
+import moment from 'moment';
 
 const DashboardContainer = Wrapped =>
   class extends React.Component {
     static propTypes = {
       userDetails: PT.object,
       fetchEvents: PT.func.isRequired,
-      fetchEventsByUserId: PT.func.isRequired,
       takenEvents: PT.array,
       isEventBeingUpdated: PT.bool,
     };
 
     constructor(props) {
       super(props);
       this.state = {
+        calendarDate: new moment(),
         filteredEvents: [],
         activeEventIds: [],
         activeEmployee: -1,
       };
     }
 
     componentDidMount() {
-      this.props.fetchEvents();
+      const { calendarDate } = this.state;
```

Comment:
Could we not call fetchEvents method instead. save duplicating code?

Reply:
Ayup. Good spot.